cleanse/cleanse.go

```diff
@@ -228,6 +233,23 @@ func Cleanse() {
 	currencies := readCurrencies("data/original/currencies.json")
 	writeJson("data/cleansed/currencies.json", currencies)
 
+	writeJson("data/cleansed/country-duties.json",
+		toObjects(readCsv("data/original/country-duties.csv"),
+			func(record map[string]string) bool {
+				return record["duty"] != "" && record["duty"] != "--"
+			},
+			func(record map[string]string) interface{} {
+				return CountryDuty{
+					CountryCode:   record["country"],
+					DeliveredDuty: record["duty"],
+				}
+			},
+			func(record map[string]string) string {
+				return record["duty"] + record["country"]
```

Comment:
this key is used to sort the data - so may be bit nicer to sort by country first: 
`record["country"] + record["duty"]`